### ISSUE TITLE
No .bot file after download

### ISSUE THREAD

**Issue by shchoy** [MicrosoftDocs/bot-docs#721]
I couldn't find the .bot file after downloading. 
Using Javascript btw.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9d5bd35d-3178-42cd-2893-3b6a4b7a94ce
* Version Independent ID: 1545fd44-d471-d947-f94d-4b5e8d6e853f
* Content: [Tutorial to create and deploy a basic bot - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tutorial-basic-deploy?view=azure-bot-service-4.0&tabs=javascript)
* Content Source: [articles/v4sdk/bot-builder-tutorial-basic-deploy.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/v4sdk/bot-builder-tutorial-basic-deploy.md)
* Service: **bot-service**
* Product: **bot-framework**
* GitHub Login: @ivorb
* Microsoft Alias: **v-ivorb**

**Comment by ross-p-smith** [comment]
No .bot file after download either - using dotnet

**Comment by ivorb** [comment]
@ross-p-smith I just created a CSharp web app bot and downloaded it, and I have a bot file. What files did you get for the download? Did you use an MSA or work account?

**Comment by ivorb** [comment]
And, I get one for Node as well... @shchoy did you use an MSA account? I didn't test it with that, and while I don't see how that would make a difference, that's the only difference I can see

**Comment by ivorb** [comment]
We're testing it with MSA now, we'll see if we can repro the issue

**Comment by ross-p-smith** [comment]
I was using work/school account.

**Comment by AskYous** [comment]
I have experienced the same thing. I am using NodeJS and my work's email account, which is an Office 365 email account. I downloaded the files but it's missing a `.bot` file.

**Comment by AskYous** [comment]
Looking in the online code editor, I see there is no .bot file available to view:
![image](https://user-images.githubusercontent.com/791996/51711145-0c3d7380-1fe0-11e9-8543-62e95f322820.png)

**Comment by DanDev33** [comment]
I was also able to create, test in Azure, and download a JS bot with the .bot file today. We found that you need to use the latest az cli tool (v 2.0.55). 
Note - With this latest version of the az cli tool, you should uninstall the botservice extension.

**Comment by DanDev33** [comment]
Closing, same issue/resolution as #735.

**Comment by ross-p-smith** [comment]
I have found a way to reproduce the missing .bot file. I am using work/school account. I have managed to download with the bot file and without the bot file. All down to one difference. I did not define the --lang as the documentation said that CSharp was default which is what I wanted.

The issue is with the **create** command.

This creates a bot file
az bot create --kind webapp --name rosmith1 --lang Csharp --location westeurope --version v4 --verbose --resource-group repro1

This **does not** create a bot file.
az bot create --kind webapp --name rosmith --location westeurope --version v4 --verbose --resource-group repro
